test/cmdlineTests.sh

```diff
@@ -277,6 +271,11 @@ printTask "Running general commandline tests..."
         # Use printf to get rid of the trailing newline
         inputFile=$(printf "%s" "${inputFiles}")
 
+        # If no files specified, assume input.sol as the default
```

Comment:
Well could also just add a non-existent filename to the affected test case. It is more clear, I just did not wanted to change these scripts when I added the test.

Reply:
Ah never mind, the scripts still search the directory for files.

Reply:
I tried that initially but it does not work. Errors out reporting that there are multiple json files specified as input.
The input file is passed separately from other options so `solc` gets `""` and your file name if you specify it in `args`.